from app.common.exceptions.repositories.base import BaseRepoError


class AttributeRepoError(BaseRepoError):
    pass
